replication_parameters.go

```diff
@@ -47,6 +47,12 @@ type ReplicationParameters struct {
 	Async            bool
 }
 
+func NewReplicationParameters() *ReplicationParameters {
```

Comment:
I removed this and the test itself, which was made redundant by tests that I added later during the dev process.